dev/tests/integration/testsuite/Magento/Catalog/_files/products_for_search.php

```diff
@@ -111,7 +116,7 @@
         ->setMetaDescription($data['meta_keyword'])
         ->setVisibility($data['visibility'])
         ->setStatus($data['status'])
-        ->setStockData(['use_config_manage_stock' => 0])
+        ->setStockData(['use_config_manage_stock' => 0, 'qty' => $data['qty']])
```

Comment:
Looks like we need to process records with `'use_config_manage_stock' => 0` without set qty
Pls create separate ticket like `Adapt use_config_manage_stock = 0 ` and specify in new ticket this fixture

Reply:
Done https://github.com/magento-engcom/msi/issues/449